package.json

```diff
@@ -0,0 +1,26 @@
+{
+  "name": "sentry-cli",
+  "version": "0.25.0",
+  "description": "A command line utility to work with Sentry. https://docs.sentry.io/hosted/learn/cli/",
+  "scripts": {
+    "test": "echo \"Error: no test specified\" && exit 1",
+    "preinstall": "INSTALL_DIR=./bin ./bin/install-sentry-cli"
```

Comment:
I couldn't see where you had the source for the script​ in the repo, which is why I copied it. If its in a different repo, you could just as eaily put the `package.json`, `bin/`, & `.npmignore` files there and that way you'd just bump the version in the package & get-cli script simultaneously in your build process.

The other alternative (which seems hackier to me, but may be easier for you guys to support) is to change the `preinstall` npm script to be 
```json
"preinstall": "INSTALL_DIR=./bin curl -sL https://sentry.io/get-cli/"
```